optionals/compat_rhs_afrf3/CfgVehicles.hpp

```diff
@@ -282,14 +282,28 @@ class CfgVehicles {
     class Helicopter_Base_F;
     class Helicopter_Base_H: Helicopter_Base_F {
         class EventHandlers;
+        class Turrets;
+    };
+    class Heli_Light_02_base_F: Helicopter_Base_H {
+        class Turrets: Turrets {
+                class CopilotTurret;
+                class MainTurret;
```

Comment:
`Heli_Light_02_base_F` does have `MainTurret` in vanilla. It's inherited from `Helicopter_Base_F` -> `Helicopter_Base_H`.